### DISCUSSION TITLE
Separate 'Stars' from 'Lists'

### DISCUSSION THREAD

**Discussion by tamir-nakar** [community/community#64839] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

To start -  think the new 'Lists' feature is a bless.
I've like 20 private repositories, and puting them in lists can make my life much easier.

for instance, i've repositories that are educational and are an output of a coures iv'e taken, alongside some other project, some are done, some are active right now.

The thing I desliked, was that **having a repo in a list means it has a star** I wish I could somehow orginize my repos in lists in a separate way fro my 'favorites' = stars. In my mind starts = a one-click solution to my most active repositories.

I also undersatnd it has a great value orginizing stars of other contributors, but as I said, I'd like to have my private way to orginize my repositories as well, without having them in the same 'stars' bucket of external repositories.

to sum up:

list item is not necessarally a star.
and a star is not necessarally in a list.

What do you think?
